classes/app.rb

```diff
@@ -7,20 +6,28 @@
 require_relative '../modules/create_book'
 require_relative '../modules/list_books'
 require_relative '../modules/list_labels'
+require './modules/create_music_album'
+require './modules/list_genre'
+require './modules/list_music_albums'
+require './modules/save'
+require 'json'
 require_relative '../json/book_json'
 require_relative '../json/label_json'
 
 class App
   include CreateBook
   include LabelList
+  include GenreList
+  include MusicAblbumList
+  include PreserveMusicAlbumGenre
   include BookJson
   include LabelJson
```

Comment:
Missing CreateMusicAlbum Module in class App, please include it.

Reply:
Done!